ocs_ci/deployment/hub_spoke.py

```
@@ -4067,7 +4067,7 @@ def create_s3_bucket_for_hypershift_oidc(
         try:
             existing_buckets = self.list_buckets()
             if any(bucket["Name"] == bucket_name for bucket in existing_buckets):
-                logger.info(f"Bucket '{bucket_name}' already exists, skipping creation")
+                logger.info(f"Bucket '{bucket_name}' already exists, reusing it")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)